recipes-openxt/xenclient/xenclient-toolstack_git.bb

```diff
@@ -60,9 +60,9 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 S = "${WORKDIR}/git"
 
 # TODO: ocamlc can figure it out in the build-system.
-CFLAGS_append = " -I${OCAML_HEADERS}"
+CFLAGS_append = " -I${ocamlincdir}"
```

Comment:
I'm getting a build error on a fresh setup with this definition as is here. Switching it back to `OCAML_HEADERS` seems to have fixed it.